exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/ColumnDefault.kt

```diff
@@ -0,0 +1,159 @@
+package org.jetbrains.exposed.sql
+
+/**
+ * Represents a default value for a column.
+ *
+ * Commonly it should be implementation of [ClientColumnDefault] or [DatabaseColumnDefault] interfaces.
+ */
+interface ColumnDefault<T>
+
+/**
+ * Represents a client-side default value for a column.
```

Comment:
please consider adding examples and with functions where these columns are used and references to the functions as well